packages/tenant-process/src/services/tenantService.ts

```diff
@@ -190,3 +216,78 @@ export function createTenantLogic({
 
   return toCreateEventTenantAdded(newTenant);
 }
+
+export async function updateVerifiedAttributeExtensionDateLogic({
+  tenantId,
+  attributeId,
+  verifierId,
+  tenant,
+}: {
+  tenantId: string;
+  attributeId: string;
+  verifierId: string;
+  tenant: WithMetadata<Tenant> | undefined;
+}): Promise<CreateEvent<TenantEvent>> {
+  assertTenantExists(tenantId, tenant);
+
+  const verifiedTenantAttribute = tenant.data.attributes
+    .filter((att) => att.type === tenantAttributeType.VERIFIED)
+    .find((att) => att.id === attributeId) as VerifiedTenantAttribute;
+
+  assertVerifiedTenantAttributeExist(
+    tenantId,
+    attributeId,
+    verifiedTenantAttribute
+  );
+
+  const tenantVerifier = verifiedTenantAttribute.verifiedBy.find(
+    (verifier) => verifier.id === verifierId
+  );
+
+  assertOrganizationVerifierExist(
+    verifierId,
+    tenantId,
+    attributeId,
+    tenantVerifier
+  );
+
+  assertExpirationDateExist(tenantId, attributeId, verifierId, tenantVerifier);
+
+  const extensionDate =
+    tenantVerifier.extensionDate ?? tenantVerifier.expirationDate;
+
+  const updatedVerifiedBy: TenantVerifier[] =
```

Comment:
Try to create a new Tenant as you can see [here](https://github.com/pagopa/interop-be-monorepo/blob/d1acaaf6d0975eb09a8b4a816cc8117634317f51/packages/tenant-process/src/services/tenantService.ts#L131)

Reply:
In addition to creating the tenant as required, I also modified the `function assertVerifiedAttributeExistsInTenant`
 to reduce the code a bit in `tenantService`